commands/inspector.go

```diff
@@ -53,6 +55,53 @@ Prints out information about filecoin process and its environment.
 		allInfo.FilecoinVersion = GetInspectorAPI(env).FilecoinVersion()
 		return cmds.EmitOnce(res, allInfo)
 	},
+	Type: AllInspectorInfo{},
+	Encoders: cmds.EncoderMap{
+		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, info *AllInspectorInfo) error {
+			sw := NewSilentWriter(w)
+
+			// Print Version
+			sw.Printf("Version:\t%s\n", info.FilecoinVersion)
+
+			// Print Runtime Info
+			sw.Printf("\nRuntime\n")
+			sw.Printf("OS:           \t%s\n", info.Runtime.OS)
+			sw.Printf("Arch:         \t%s\n", info.Runtime.Arch)
+			sw.Printf("Version:      \t%s\n", info.Runtime.Version)
+			sw.Printf("Compiler:     \t%s\n", info.Runtime.Compiler)
+			sw.Printf("NumProc:      \t%d\n", info.Runtime.NumProc)
+			sw.Printf("GoMaxProcs:   \t%d\n", info.Runtime.GoMaxProcs)
+			sw.Printf("NumGoRoutines:\t%d\n", info.Runtime.NumGoRoutines)
+			sw.Printf("NumCGoCalls:  \t%d\n", info.Runtime.NumCGoCalls)
+
+			// Print Disk Info
+			sw.Printf("\nDisk\n")
+			sw.Printf("Free:  \t%d\n", info.Disk.Free)
+			sw.Printf("Total: \t%d\n", info.Disk.Total)
+			sw.Printf("FSType:\t%s\n", info.Disk.FSType)
+
+			// Print Memory Info
+			sw.Printf("\nMemory\n")
+			sw.Printf("Swap:   \t%d\n", info.Memory.Swap)
+			sw.Printf("Virtual:\t%d\n", info.Memory.Virtual)
+
+			// Print Config Info
+			sw.Printf("\nConfig\n")
```

Comment:
BLOCKING: this should be printed last as it does not match the format of the rest.